src/resources/elements/primeDesignSystem/pform-input/pform-input.html

```diff
@@ -0,0 +1,27 @@
+<template>
```

Comment:
I always appreciate some comments in HTML to have a visual separation of the different parts.
Here I see
1. Label
2. Label description
3. Actual input
4. Helper message
5. Feedback message

Totally fine, not to add though.